src/Sql/Sql.Sdk/Generated/Models/ServerBlobAuditingPolicy.cs

```diff
@@ -321,7 +321,7 @@ public ServerBlobAuditingPolicy(BlobAuditingPolicyState state, string id = defau
         /// SELECT on SCHEMA::mySchema by public
         ///
         /// For more information, see [Database-Level Audit
-        /// Actions](https://learn.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-actions)
+        /// Actions](https://docs.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-actions)
```

Comment:
The docs URL has been updated to learn.microsoft.com instead of docs.microsoft.com. Please revert the change and re-generate the code.